server.js

```diff
@@ -31,3 +31,12 @@ exclusionDates.saveExclusionDays()
 
     module.exports = app;
   });
+
+if (process.env.NODE_ENV === 'test') {
+  const app = hof(settings);
+  app.use((req, res, next) => {
+    res.locals.htmlLang = 'en';
+    next();
+  });
+  module.exports = app;
+}
```

Comment:
Not sure why line 24-33 has a .then block, not sure what promise it's resolving since the saveExclusionDays function is in a try/catch block and it seems to me that the errors are being handled there inside the catch block, but i'm not sure.

Below is code I have added that seems to run the app fine and the ui-test fine with the added benefit that because I am not setting node_env to test, it means that I can run the ui-tests while the app is running locally.

```
try {
  exclusionDates.saveExclusionDays();
} catch (e) {
  console.log(e);
}

const app = hof(settings);

app.use((req, res, next) => {
  res.locals.htmlLang = 'en';
  next();
});

module.exports = app;
```